robottelo/performance/graph.py

```diff
@@ -93,15 +47,15 @@ def generate_stacked_line_chart_raw(time_result_dict, head, filename):
     stackedline_chart.render_to_file(filename)
 
 
-def generate_line_chart_raw(time_result_dict, head, filename):
-    """Generate Normal Line chart for raw data of ak/att/del/reg
+def generate_line_chart_raw(time_result_dict, head, filename, line_chart):
+    """Common function for line charts used by Candlepin and Pulp tests
```

Comment:
This docstring doesn't tell me what the function does. Don't tell me that this function contains common functionality - of course it does. It's very common for a function to do that. Do tell me what this function does.